pytorch-mobilenet-image/README.md

```diff
@@ -77,3 +81,25 @@ Executed graph inference
    4.) [950](10.4899)orange
    5.) [953](9.4834)pineapple, ananas
 ```
+
+- Case 2: Apply named model feature
+> requirement wasi-nn >= 0.5.0 and WasmEdge-plugin-wasi_nn-(*) >= 0.13.4
+
+
+```bash
+wasmedge --dir .:. --nn-preload demo:PyTorch:CPU:mobilenet.pt wasmedge-wasinn-example-mobilenet-image-named-model.wasm demo input.jpg
```

Comment:
Can you explain what the `demo:PyTorch:CPU:mobilenet.pt` name means?

Reply:
This is defined in the named model feature: `--nn-preload`
Ref: https://github.com/WebAssembly/wasi-nn/issues/36

Reply:
But the encoding and target are determined by the API calls in the code. What if the information in the name conflicts with the settings in the code? Why do we have to repeat ourselves here?

Reply:
In fact, `load_by_name` doesn't have parameters to specify the target and encoding of building the graph. Similarly, you can't assign a name to a model loaded by `load_by_files` ( or `bytes`) (loading a model from WebAssembly). These two mechanisms, IIUC from the API definition, there don't have interoperability. By the way, there's a scenario where `--nn-preload` multiple models with conflicting names can occur (our implementation the later one will overwrite the earlier one).

Reply:
In fact, `wasi_ephemeral_nn::load_by_name` doesn't have parameters to specify the target and encoding of building the graph. Similarly, you can't assign a name to a model loaded by `load_by_files` ( or `bytes`) (loading a model from WebAssembly). These two mechanisms, IIUC from the API definition, there don't have interoperability. By the way, there's a scenario where `--nn-preload` multiple models with conflicting names can occur (our implementation the later one will overwrite the earlier one).